phase1-cli/src/bin/prepare_phase2.rs

```diff
@@ -69,8 +69,8 @@ fn prepare_phase2<E: Engine + Sync>(opts: &PreparePhase2Opts) -> Result<()> {
     // Deserialize the accumulator
     let current_accumulator = Phase1::deserialize(
         &response_readable_map,
-        UseCompression::Yes,
-        CheckForCorrectness::Full,
+        UseCompression::No,
+        CheckForCorrectness::No,
```

Comment:
if we've already checked the output of Phase1, then we don't need to check for correctness, which makes the preparation step faster. This should be re-enabled if we did not check that the output of Phase 1 is in the appropriate prime-order group.